backend/app/models/enrollment.rb

```diff
@@ -169,6 +171,50 @@ def copy(current_user)
     copied_enrollment
   end
 
+  def team_members_json
```

Comment:
Strange to render this field as a json in a CSV, but why not for a first version.

Reply:
agreed